data_steward/analytics/table_metrics/weekly_scripts/concept_success_rate.py

```diff
@@ -47,7 +47,7 @@
 
 import matplotlib.pyplot as plt
 
-DATASET = ''
+DATASET = '''
```

Comment:
Is this intentional?

Reply:
no - not sure how that got in there. i'll be sure to change it in my next PR. does this affect the script at all?

Reply:
@nishanthpp93 I am making updates to this anyways; I will make another PR sometime later today.